environment.yml

```diff
@@ -2,7 +2,7 @@ name: clisops
 channels:
  - conda-forge
 dependencies:
- - python >=3.8
+ - python >=3.8,<3.12
```

Comment:
Why < 3.12 ?

Reply:
ReadTheDocs will try to build the documentation with the highest available version. CLISOPS only supports Python >=3.8,<3.12 at the moment.